src/phoenix/server/api/types/Span.py

```diff
@@ -105,6 +113,7 @@ def to_gql_span(row: "Series[Any]") -> Span:
     Converts a dataframe row to a graphQL span
     """
     attributes = _extract_attributes(row).to_dict()
+    events: List[SpanEvent] = to_gql_span_events(row["events"] or "[]")
```

Comment:
Could make `to_gql_span_event` more "atomic". Also, `data["events"]` is always a list (not str) because how we encode and decode the `jsonl`.
```suggestion
    events: List[SpanEvent] = list(map(to_gql_span_event, row["events"]))
```

Reply:
I think the random data might be outdated. Fixed